kratos/processes/find_intersected_geometrical_objects_process.cpp

```diff
@@ -212,7 +204,9 @@ void FindIntersectedGeometricalObjectsProcess::Execute()
 
 void FindIntersectedGeometricalObjectsProcess::ExecuteInitialize()
 {
-    GenerateOctree();
+    if (mrModelPartIntersected.NumberOfNodes() > 0) {
```

Comment:
Although so far we assume that we have the complete skin model part in each node, I'd check the global number of nodes in here.

Reply:
You mean checking if GlobalNumberOfNodes > 0?. 

Note that his would make every partition (even the empty partitions) enter in GenerateOctree() which would lead to the error reported initially

Sorry if I misunderstood you

Reply:
My bad, I thought you were checking the skin model part nodes. This is OK. Forget about my comment.